### ISSUE TITLE
matplotlib code cruft

### ISSUE THREAD

**Issue by cnorthway** [accessibility-oscilloscope/signal-processing#1]
if we're going to remove it, go all the way. commenting out the import isn't great
there's a whole bunch of plotting code left too, thankfully it isn't trying to run

if we want it for testing, maybe some kind of modularity would let us see the results without using it in the main processing stage

**Comment by nicholasmullikin** [comment]
Yeah that's true. I wanted to get an MVP working before we go through and clean up the code since I haven't gotten the single period code working yet.
